workflows/data_pipelines/metadata/cc/task_functions.py

```diff
@@ -98,26 +126,23 @@ def create_metadata_concollective_json():
         json.dump(metadata_json, json_file)
 
 
-def upload_json_file_to_minio():
+@task()
+def upload_json_to_minio():
+    context = get_current_context()
+    ti = context["ti"]
+
     minio_client.send_files(
         list_files=[
-            {
-                "source_path": METADATA_CC_TMP_FOLDER,
-                "source_name": "metadata-cc-kali.json",
-                "dest_path": METADATA_CC_MINIO_PATH,
-                "dest_name": "cc_kali.json",
-            }
+            File(
+                source_path=METADATA_CC_TMP_FOLDER,
+                source_name="metadata-cc-kali.json",
+                dest_path=METADATA_CC_MINIO_PATH,
+                dest_name="cc_kali.json",
+                content_type=None,
```

Comment:
Since this is optional, I might just not mention it.

Reply:
`File` definition is a `class File(TypedDict)` so a dictionnary.
If we don't setup `content_type` it will raise an error if we do `file["content_type"]`.
Unfortunately `TypedDict` does not allow for default values.

I was thinking about making it a dataclass but I noticed is used everywhere so I felt like it could wait for the bigger refactoring. I was afraid of creating bugs to other DAGs haha.